GrowthBook/GrowthBook.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
-    <Authors>Ben Whatley</Authors>
+    <Authors>Ben Whatley,Norhaven</Authors>
```

Comment:
I would usually only put recognized contributors of the Github repository here

Reply:
Fair enough, removed. :)